Website/DesktopModules/Hotcakes/Core/Models/AffiliateViewModel.cs

```diff
@@ -166,9 +166,12 @@ public string ToJson()
         [RegularExpression(@"[-\w]*", ErrorMessage = "Only alpha-numeric and '-', '_' characters are allowed")]
         public string ReferralAffiliateId { get; set; }
 
+        public bool IsTrue => true;
```

Comment:
Please change this to a standard property.  The build process won't recognize this version of C# (right now).

Reply:
@erw13n I don't see your update on this one yet?

Reply:
Understood, I'll update it.

Reply:
I already update it, its on commit of "Update the other Viewset" below.